pages/api/question.ts

```diff
@@ -105,6 +105,7 @@ export type SalesTaxModel = {
   personName: string;
   image1: string;
   multipleAnimals: string,
+  numberOfToys: number,
```

Comment:
Not sure why this is here. You probably need to merge latest main into this branch?

Reply:
makes sense. I removed.